modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/service/impl/SXPBlueprintLocalServiceImpl.java

```diff
@@ -325,6 +347,13 @@ private void _validate(
 	@Reference
 	private AssetSubtypeIdentifierBuilder _assetSubtypeIdentifierBuilder;
 
+	private final List<String> _collectionProviderTypes = new ArrayList<>(
+		Arrays.asList(
+			"com.liferay.blogs.model.BlogsEntry",
+			"com.liferay.document.library.kernel.model.DLFileEntry",
```

Comment:
Hi @BryanEngler,

I was wondering if you had a better idea on how to these types should be checked. Maybe a Constants class?